gwsumm/data/mathutils.py

```diff
@@ -194,6 +194,9 @@ def get_with_math(channel, segments, load_func, get_func, **ioargs):
         # apply math to this channel
         cmath = singleops[names[0]]
         for op_, val_ in cmath:
+            if (op_ is operator.add or op_ is operator.sub) and \
```

Comment:
```suggestion
            if op_ in {operator.add, operator.sub} and \
```

Reply:
thanks @duncanmmacleod, I took care of this in the latest commit. I also took the step checking that `val_` is an instance `numbers.Number` rather than just `float`.